packages/api/src/db/block.ts

```diff
@@ -191,6 +192,25 @@ export interface TeaserGridBlock {
   numColumns: number
 }
 
+export interface FlexItemAlignment {
```

Comment:
I'm a bit confused by the name. Why does it have `Item` in it. I think `FlexAlignment` would be better here.